MakerTracker/ClientApp/src/app/services/auth/auth.service.ts

```diff
@@ -28,8 +28,10 @@ export class AuthService {
   // from: Convert that resulting promise into an observable
   isAuthenticated$ = this.auth0Client$.pipe(
     concatMap((client: Auth0Client) => from(client.isAuthenticated())),
-    tap(res => this.loggedIn = res)
+    tap(res => this.loggedIn = res),
+    tap(res => this.isLoggedIn$.next(res))
   );
+  isLoggedIn$ = new BehaviorSubject<boolean>(false);
```

Comment:
I had to do this because subscribing to `isAuthenticated$` immediately unsubscribes after it returns.